scripts/zkai_native_attention_mlp_rmsnorm_adjacent_layout_gate.py

```
@@ -0,0 +1,581 @@
+        "adjacent_label_span_typed_bytes",
+        "variants",
+        "interpretation",
+        "non_claims",
+        "validation_commands",
+        "mutation_results",
+        "payload_commitment",
+    }
+    if set(payload) != required:
+        raise AdjacentLayoutGateError("payload key drift")
+    if payload["schema"] != SCHEMA or payload["decision"] != DECISION or payload["result"] != RESULT:
+        raise AdjacentLayoutGateError("payload metadata drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)